pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/store/TextIndexUtils.java

```diff
@@ -43,9 +43,9 @@ private TextIndexUtils() {
 
   static void cleanupTextIndex(File segDir, String column) {
     // Remove the lucene index file and potentially the docId mapping file.
-    File luceneIndexFile = new File(segDir, column + Indexes.LUCENE_TEXT_INDEX_FILE_EXTENSION);
+    File luceneIndexFile = new File(segDir, column + Indexes.LUCENE_V9_TEXT_INDEX_FILE_EXTENSION);
```

Comment:
Should we also delete v8 indices?

Reply:
:+1:

Reply:
I feel we should delete v8 index after next version? In case user want to downgrade/rollback.

Reply:
Actually this is explicitly delete, then we should delete both.